### DISCUSSION TITLE
Backup thumbnails

### DISCUSSION THREAD

**Discussion by videosmith** [k-ivan/visual-bookmarks-chrome#87] [Q&A]
Thank you for a great extension! Was using Opera until I switched to Arm processors :/

Question: If I wanted to port my bookmarks to another computer, is it possible to move your bookmark/thumbnail database to avoid re-scanning each website? In Opera, there are favicons files that store their url/thumb data....

Currently using Chromium on a Debian distribution (Raspberry Pi Bullseye 64).

**Comment by k-ivan** [comment]
Hello!

Unfortunately, at the moment there is no way to transfer thumbnails using the extension itself.

But you can try moving your entire browser profile to another computer. I'm not sure if this will work as I don't have similar experience. But maybe this will help...
The link is for instructions for Windows, but I think it will not be difficult to look ways for Linux by analogy.(https://www.winhelponline.com/blog/transfer-chrome-profile-another-computer/)

**Note**: If you try, then **do not reset the cache**, some articles advise, since the volume of your folder may be large, but you cannot delete the cache, otherwise the thumbnails will be deleted along with it.

**Comment by videosmith** [comment]
Thank you. I will try that.
